spec/requests/records_spec.rb

```diff
@@ -60,7 +60,7 @@
         end
 
         it "paginates results according to per_page value", :multi_records do
-          get "/records?per_page=10"
+          get "/records?page=2&per_page=10"
```

Comment:
This works but it's redundant because we don't need to do "get" any more (it's on my TODO to update it). Instead run_test! takes care for us and we update the context.   We have some example of this in other tests.